core/src/main/resources/swagger/api-docs.yaml

```
@@ -5625,7 +5625,27 @@ components:
             Sam.
         managedAppCoordinates:
           $ref: '#/components/schemas/AzureManagedAppCoordinates'
+        members:
+          description: The list of members to add to the billing project
+          type: array
+          items:
+            $ref: '#/components/schemas/ProjectAccessUpdate'
+        inviteUsersNotFound:
+          description: If true, invite the users to Terra if they are not registered in the system. They will be given access to the project
+          type: boolean
       description: ""
+    ProjectAccessUpdate:
+        required:
+          - email
+          - role
+        type: object
+        properties:
+          email:
+            description: the email of the member to modify access for
+            type: string
+          role:
+            description: the role to modify for the specified user
```

I found this description a little unclear. Maybe something like:

```suggestion
            description: the permissions to grant to the specified user
```

---

I had it this way because this same model will also be used to revoke roles from members in the near future